### PR TITLE
Homework3: Alexey Kozhan

### PR DESCRIPTION
**Pull request by AlexKozhan**
Изменен файл readme.md

### CODE REVIEW

**Comment by Stml89** [comment]
Please correct PR name.

JFYI:
Start using English for commit messages and comments

**Review by Stml89 — CHANGES_REQUESTED** [review]

File: .github/workflows/ithub-actions-demo.yml
[the remark this answers is not in the document]
Comment (AlexKozhan) [reply .github/workflows/ithub-actions-demo.yml:1]: Removed file ".github/workflows/ithub-actions-demo.yml"
Changed PR name

**Comment by AlexKozhan** [comment]
Did i succeed?

**Review by Stml89 — APPROVED** [review]
Approved

**Comment by Stml89** [comment]
Now you can merge it to the master branch

**Pull request merged**